src/elements/content-sidebar/activity-feed/version/Version.scss

```diff
@@ -1,11 +1,12 @@
 @import '../../../common/variables';
+@import '../mixins';
 
 .be {
     .bcs-Version {
         display: flex;
         align-items: center;
         justify-content: space-between;
-        width: 100%;
+        margin: 0 $bcs-ActivityFeedItem-padding-horizontal;
```

Comment:
Can rename the variable to `$bcs-ActivityFeedItem-spacing-horizontal` to avoid margin/padding naming confusion?